app/models.py/Converter.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+
+class Converter(models.Model):
```

Comment:
only class in python we use camelCase with them 
filename should follow snake case for best practisce
https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html